src/libsync/syncengine.cpp

```diff
@@ -1023,18 +1032,23 @@ void SyncEngine::slotItemCompleted(const SyncFileItem &item, const PropagatorJob
     emit itemCompleted(item, job);
 }
 
-void SyncEngine::slotFinished()
+void SyncEngine::slotFinished(bool success)
 {
     _anotherSyncNeeded = _anotherSyncNeeded || _propagator->_anotherSyncNeeded;
 
+    qWarning() << success << _discoveryMainThread->_dataFingerprint;
+    if (success) {
```

Comment:
If it fails (say, the connection was closed)  we want the next sync to continue the restoration.

I need to remove the qWarning.